tests/vm_test.go

```diff
@@ -428,6 +428,34 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			return updatedVM
 		}
 
+		table.DescribeTable("cpu/memory in requests/limits should allow", func(cpu, request string) {
+			vm := tests.NewRandomVirtualMachine(
+				tests.NewRandomVMIWithEphemeralDisk(
+					cd.ContainerDiskFor(cd.ContainerDiskCirros),
+				), false,
+			)
+			vm.APIVersion = "kubevirt.io/" + v1.ApiStorageVersion
+			vm.Spec.Template.Spec.Domain.Resources.Limits = make(k8sv1.ResourceList)
+			vm.Spec.Template.Spec.Domain.Resources.Requests[corev1.ResourceCPU] = resource.MustParse("222")
+			vm.Spec.Template.Spec.Domain.Resources.Limits[corev1.ResourceCPU] = resource.MustParse("222")
+			vm.Spec.Template.Spec.Domain.Resources.Requests[corev1.ResourceMemory] = resource.MustParse("2222222")
+			vm.Spec.Template.Spec.Domain.Resources.Limits[corev1.ResourceMemory] = resource.MustParse("2222222")
+
+			jsonBytes, err := json.Marshal(vm)
+			Expect(err).NotTo(HaveOccurred())
+
+			jsonString := strings.Replace(string(jsonBytes), "\"222\"", cpu, -1)
```

Comment:
that part is a little bit tricky to understand, but should work.

Reply:
something like this would make it perhaps a little more obvious from a code readability standpoint.

```
var cpuMatchOld = `"222"`
jsonString := strings.ReplaceAll(string(jsonBytes), cpuMatchOld, cpu)
```

the same pattern could be used for memory below